src/state/initialize/functions/generateTextTriangleParams.js

```diff
@@ -10,8 +10,8 @@ export default (function generateTextTriangleParams(store, viz_area) {
     row: {},
     col: {},
     draw: {
-      row: [],
-      col: [],
+      row: false,
+      col: false,
```

Comment:
Should we align the data structure here for better readability?

Reply:
You mean set it to an empty array? It could be nice but we would need to modify the code elsewhere so it looks for an empty array instead of `false`. So for now we should keep it consistent with the way it's been done, and we can do that in another PR if you want.